frontend/public/components/_route.scss

```diff
@@ -15,3 +11,26 @@
 .co-m-route-tls-reveal__btn {
   padding-right: 0;
 }
+
+.co-m-route__ingress-status-section {
+  .co-m-route__ingress-status {
+    padding-bottom: 40px;
+
+    .co-m-table-grid {
```

Comment:
This is not really BEM. You shouldn't have this nesting. `.co-m-table-grid` styles shouldn't change based on the page they're on.

Reply:
All of the other tables in the console have some margin. Let's be consistent and not override table styles on just one page. If we want to change it, we should change it everywhere.

For example,

![screen shot 2018-06-22 at 2 23 02 pm](https://user-images.githubusercontent.com/1167259/41792767-f2fec002-7627-11e8-8124-e05914d085d4.png)

Reply:
(Just a quick comment that we are fixing the awkward wrapping in these tables under #149)

Reply:
> All of the other tables in the console have some margin. Let's be consistent and not override table styles on just one page. If we want to change it, we should change it everywhere.

This was a request from @beanh66 , I believe due to the fact that Conditions go along with the current Router and there maybe multiple Routers repeated within the section.   Should each Router actually be it's own section (separated by a section lines)?  If so, then I can see it being like the other sections on the page Ie. "TLS Settings".